docs/spmd.md

```diff
@@ -46,8 +46,8 @@ import numpy as np
 import torch
 import torch_xla.core.xla_model as xm
 import torch_xla.runtime as xr
-import torch_xla.experimental.xla_sharding as xs
-from torch_xla.experimental.xla_sharding import Mesh
+import torch_xla.distributed.spmd.xla_sharding as xs
```

Comment:
What do you think about making the import path just `import torch_xla.distributed.spmd as xs`? Not sure if `xs` is still the right abbreviation in that case (maybe we can think of it as `x`la `s`pmd), but it would make the `spmd` package the single entrypoint in userspace.

Reply:
Good point, let's do that.